Extensions/Today/TodayViewController.swift

```diff
@@ -10,6 +10,7 @@ import SnapKit
 private let log = Logger.browserLogger
 
 private let privateBrowsingColor = UIColor(colorString: "CE6EFC")
```

Comment:
Why no `TodayViewControllerUX` struct like we do for other components?

Example https://github.com/mozilla/firefox-ios/blob/master/Extensions/ShareTo/ShareViewController.swift#L29